io.home-assistant.awning/manifest.tt

```diff
@@ -0,0 +1,22 @@
+abstract class @io.home-assistant.awning
+#_[thingpedia_name="Awning (Cover)"]
+#_[description="Interface for Home Assistant's Awning (Cover)."]
+#[license="CC-0"]
+#[license_gplcompatible=true]
+#[subcategory="home"]
+{
+  monitorable query state(out state : Enum(open,closed))
+  #_[canonical="awning state"]
+  #_[confirmation="the state of your awning"]
```

Comment:
I realize it might be late to tell you, but if you want to support referring to devices by name, the confirmation string should say `$__device` instead of `your awning`, as in "the state of $__device`.